markdown/src/test/java/com/m2mobi/basemarkdown/rule/ListRuleTest.java

```diff
@@ -111,7 +110,7 @@ public void shouldNotCreateNestedList() {
 		strings.add("- Number 1");
 		strings.add("- Number 2");
 		assertTrue(mListRule.conforms(strings));
-
+		MarkdownList list = (MarkdownList) mListRule.toMarkdownItem(strings);
```

Comment:
ah I forgot to remove this line before pushing. Its debug only so it can be removed, my bad

Reply:
Removed it